app/controllers/patients_controller.rb

```diff
@@ -324,12 +336,11 @@ def update_status
         next unless params[:apply_to_group_cm_only_date].present?
```

Comment:
This field no longer exists after the change so I think we can remove this and the related code below.

Reply:
Fixed in recent commit.